pillar.example

```diff
@@ -8,10 +8,13 @@ drbd:
   ## Configure DRBD in pacemaker cluster
   #with_ha: false
 
-  ## Pirmary node when promoting DRBD
+  ## Primary node when promoting DRBD
   ## TODO: Only support single primary currently
   promotion: "drbd-node1"
 
+  ## Virtual IP for the NFS group
+  virtual_ip: "192.168.10.200"
```

Comment:
I don't think we should specify this here. It is already coming from the `cluster.sls` pillar file, in the `parameters` fieldn

Reply:
I think you don't need to define the `virtual_ip` here, could reuse `data.virtual_ip` from cluster file for `primitive rsc_vip_nfs IPaddr2`

Reply:
you are right. It is set to `{{ grains['drbd_cluster_vip'] }}` there correctly.